### PR TITLE
feat: make configuration of privacy policy more flexible

### PR DESCRIPTION
**Pull request by rokroskar**
Related to https://github.com/SwissDataScienceCenter/renku-ui/issues/2954

/deploy #persist renku-ui=main extra-values=ui.client.privacy.page.enabled=true

### CODE REVIEW

**Review by leafty — COMMENTED** [review]
Chart changes look good :+1:

Should the documentation also contain the procedure to enable the "Terms and Conditions" required action on Keycloak?

Note: waiting for the KC theme image to be ready before greenlighting.

File: helm-chart/renku/values.yaml [hunk 1]
```diff
@@ -295,7 +295,7 @@ keycloakx:
     enabled: false
   extraInitContainers: |
     - name: theme-provider
-      image: renku/keycloak-theme:4.1.3
+      image: ciyerr/renku-theme:latest
```
Comment (leafty) [inline helm-chart/renku/values.yaml:298]:
```suggestion
      image: renku/keycloak-theme:4.1.5
```

**Review by leafty — APPROVED** [review]

**Comment by rokroskar** [comment]
@ciyer now I'm confused - I thought we decided against requiring Terms&Conditions approval on login? In favor of just informing?

**Comment by ciyer** [comment]
> @ciyer now I'm confused - I thought we decided against requiring Terms&Conditions approval on login? In favor of just informing?

This is correct, and this is what the PR does. I think @leafty changed the configuration in the Keycloak admin to see how much effort it is to require approval (not much, just flip two switches).

**Comment by leafty** [comment]
> Got it - can we just have one sentence somewhere pointing to keycloak docs?

We can't, our Keycloak is too out-of-date :see_no_evil: https://www.keycloak.org/documentation-archive.html

We could merge as is, and add that documentation later; I don't think it's worth delaying things here.

**Comment by rokroskar** [comment]
Ok lets leave as-is for now. It's 99% likely a non-issue.

**Pull request merged**